src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -0,0 +1,97 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+  padding-bottom: 150px;
+  padding-left: 75px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 78px;
+    padding-left: 0;
+    padding-top: 50px;
+  }
+}
+
+.allTextWrap {
+  grid-column-end: 11;
+  grid-column-start: 2;
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  margin: 0;
+  padding-bottom: 54px;
+}
+
+.number {
+  font-size: 72px;
+  height: 94px;
+  margin-bottom: 5px;
+  margin-right: 110px;
+  width: 250px;
+}
+
+.statement {
+  font: var(--font-body-medium);
+  height: 52px;
+  margin-right: 110px;
+}
+
+@media (--tablet-and-down) {
+  .statement {
+    font: var(--font-body-medium);
+    height: 52px;
+    margin: 0;
+    margin-bottom: 40px;
+    margin-right: 85px;
```

Comment:
It's redundant to set the margin properties with the shorthand and the immediately again with the individual properties for each direction. Please either use one style or the other, but don't mix the two, since the shorthand is first setting the bottom margin to 0 and then the margin-bottom property is immediately overriding it.

Reply:
👍 Deleted margin 0.